pkg/sql/subscriber.go

```diff
@@ -20,18 +20,21 @@ var (
 type SubscriberConfig struct {
 	ConsumerGroup string
 
-	// PollInterval is the interval to wait between subsequent SELECT queries, if no more messages were found in the database.
+	// PollInterval is the interval to wait between subsequent SELECT queries, if no more messages were found in the database (Prefer using the BackoffManager instead).
 	// Must be non-negative. Defaults to 1s.
 	PollInterval time.Duration
 
-	// ResendInterval is the time to wait before resending a nacked message.
+	// ResendInterval is the time to wait before resending a nacked message (Prefer using the BackoffManager instead).
```

Comment:
It seems the resend internal is not used in BackoffManager. :)

Reply:
That's right. Interesting that there's a sleep inside the `transaction` again in `sendMesage` might be worth looking at changing this to avoid it.
Most likely we should rollback and try again everything not just the sendMessage.